pkg/oc/admin/router/router.go

```diff
@@ -436,21 +435,21 @@ func generateSecretsConfig(cfg *RouterConfig, namespace string, defaultCert []by
 	return secrets, volumes, mounts, nil
 }
 
-func generateProbeConfigForRouter(cfg *RouterConfig, ports []kapi.ContainerPort) *kapi.Probe {
+func generateProbeConfigForRouter(path string, cfg *RouterConfig, ports []kapi.ContainerPort) *kapi.Probe {
 	var probe *kapi.Probe
 
 	if cfg.Type == "haproxy-router" {
 		probe = &kapi.Probe{}
-		healthzPort := defaultHealthzPort
+		probePort := defaultStatsPort
 		if cfg.StatsPort > 0 {
-			healthzPort = cfg.StatsPort
+			probePort = cfg.StatsPort
```

Comment:
@rajatchopra @pravisankar does this make it clearer? removing the reference to the healthzPort in favour of a more generic term

Reply:
+1